src/commands/shell.ts

```diff
@@ -0,0 +1,120 @@
+import { Command } from "../command";
+import { start } from "repl";
+import { yellow, red, bold } from "cli-color";
+import * as admin from "firebase-admin";
+import { getFirebaseConfig } from "../functionsConfig";
+import { getAccessToken } from "../api";
+import { getRefreshToken } from "../auth";
+import { requireAuth } from "../requireAuth";
+import { FirebaseError } from "../error";
+import { createContext, runInContext, Context, Script } from "vm";
+import { readFileSync, existsSync } from "fs";
+import fetch, { Response } from "node-fetch";
+import { DocumentSnapshot, Firestore, QuerySnapshot } from "@google-cloud/firestore";
+import api = require("../api");
```

Comment:
Can you import the other 2 values needed in line 6 above where `api` is being `import`'d?

Reply:
Done.